docs/README.md

```diff
@@ -1,12 +1,10 @@
 # nf-core/methylseq: Documentation
 
-The nf-core/methylseq documentation is split into the following files:
+The nf-core/methylseq documentation is split into the following pages:
 
-1. [Installation](https://nf-co.re/usage/installation)
-2. Pipeline configuration
-    * [Local installation](https://nf-co.re/usage/local_installation)
-    * [Adding your own system config](https://nf-co.re/usage/adding_own_config)
-    * [Reference genomes](https://nf-co.re/usage/reference_genomes)
-3. [Running the pipeline](usage.md)
-4. [Output and how to interpret the results](output.md)
-5. [Troubleshooting](https://nf-co.re/usage/troubleshooting)
+* [Usage](usage.md)
+  * An overview of how the pipeline works, how to run it and a description of all of the different command-line flags.
```

Comment:
```suggestion
  * An overview of how the pipeline works, how to run it and 
  
* [Parameters](https://nf-co.re/methylseq/parameters)
  * A description of all of the different command-line flags.
```

Reply:
```suggestion
  * An overview of how the pipeline works, how to run it and 
  [Paramters](../nextflow_schema.json)
  * A description of all of the different command-line flags.
```
See comment above.

Reply:
We can't touch this entire file as it will cause the linting to fail. Needs to be addressed in the template: https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/docs/README.md